src/components/Pagination.jsx

```diff
@@ -63,7 +66,7 @@ function Pagination({
           <li
             key={key}
             className="paginationItem"
-            aria-current="false" // change this line to highlight a current page.
+            aria-current={currentPage === pageNumber ? 'page' : false} // change this line to highlight a current page.
```

Comment:
The "aria-current" attribute expects a string and not a Boolean. Alternatively, the [logical AND(`&&`)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND) operator can be used to omit the else(`:`) block.